senza/manaus/constants.py

```diff
@@ -0,0 +1,11 @@
+ELB_REGION_HOSTED_ZONE = {"us-east-1": "Z35SXDOTRQ7X7K",
```

Comment:
This is far from ideal, but I couldn't find a good way to get the hosted zone of the ELBs for an arbitrary region, I only found the table in a documentation page (I should probably leave a comment here with a link to the documentation)